product-service/fillTable.sql

```diff
@@ -0,0 +1,52 @@
+create extension if not exists "uuid-ossp";
+
+--Create table products
+create table products (
+	id uuid not null default uuid_generate_v4() primary key,
+	title text not null,
+	description text,
+	price integer
+);
+
+--Create table stocks
+create table stocks (
+	product_id uuid not null primary key,
+	count integer
+);
+
+--Connect tables by id column 
+alter table products add column stocks_product_id uuid references stocks(product_id);
+
+
+--Insert data into products table
+insert into products (title, description, price) values ('Product1','Some description',100);
+insert into products (title, description, price) values ('Product2','Some description',150);
+insert into products (title, description, price) values ('Product3','Some description',200);
+insert into products (title, description, price) values ('Product4','Some description',250);
+insert into products (title, description, price) values ('Product5','Some description',300);
+insert into products (title, description, price) values ('Product6','Some description',350);
+insert into products (title, description, price) values ('Product7','Some description',400);
+insert into products (title, description, price) values ('Product8','Some description',450);
+insert into products (title, description, price) values ('Product9','Some description',500);
+insert into products (title, description, price) values ('Product10','Some description',550);
```

Comment:
JFYI you can use one statement to add multiple rows
```suggestion
insert into products (title, description, price) values
('Product1','Some description',100),
('Product2','Some description',150),
('Product3','Some description',200),
('Product4','Some description',250),
('Product5','Some description',300),
('Product6','Some description',350),
('Product7','Some description',400),
('Product8','Some description',450),
('Product9','Some description',500),
('Product10','Some description',550);
```